### DISCUSSION TITLE
Change statusline icon

### DISCUSSION THREAD

**Discussion by Maritsu** [NvChad/NvChad#2646] [Q&A] [answered]
This is a really minor change I want to make, though I figured I'd still ask about it. How could I change the Vim icon that's shown next to the mode name on the status line?
![image](https://github.com/NvChad/NvChad/assets/66056347/5850af61-2b0b-4674-b2fc-0a1fe7a03e48)

**Comment by siduck** [comment]
https://nvchad.com/docs/config/nvchad_ui

**Reply by wooshings** [reply]
this feels completely unhelpful. the documentation for statusline modules, which i have to assume is the correct place to be searching, is almost empty.

**Reply by siduck** [reply, marked as answer]
@wooshdude :h nvui.stl
